webserver/Controllers/CompanyController.cs

```diff
@@ -40,6 +40,10 @@ public class CompanyController : ControllerBase {
     [HttpGet]
     public async Task<IActionResult> ReadCompanies(int? offset, int? limit, string? Country, string? State, string? City, string? sort) {
 
+        if(limit<1){
+            return BadRequest("Results amount are limited to less than 1");
```

Comment:
This message is confusing.

```suggestion
            return BadRequest("Limit parameter must be a natural number greater than 0");
```

Reply:
Done